Controller/ConfigController.php

```diff
@@ -110,6 +110,20 @@ public function newAction(Request $request, $adapter)
                     $message->success(sprintf('The config %s is saved', $data->getName()));
                 }
 
+                if($data->getAdapter() == "twitter")
```

Comment:
Can you think about the code; which makes it more dynamically so new adapters can be added easily instead of copying and changing some values?